modules/retail/pages/retail-install-unified.adoc

```diff
@@ -60,13 +67,29 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
 Review the settings and click btn:[Install].
 
 
-
-When the installation procedure has finished, you can check that all the required modules are added.
-Use the [command]``SUSEConnect --status-text`` command at a command prompt.
-For {smr}, the expected modules are:
-
-* {sles} Basesystem Module
-* Python 2 Module
-* Server Applications Module
-* SUSE Manager Proxy Module
-* SUSE Manager Retail Branch Server Module
+.Procedure: Configuration and registering of the Branch Server
+. Create an activation key based on the [systemitem]``SLE-Product-SUSE-Manager-Retail-Branch-Server-4.0-Pool`` base channel.
+For more information about activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[Creating Activation Key].
+. From the [guimenu]``Child Channels`` listing select the recommended
+channels by clicking the ``include recommended`` icon:
++
+* SLE-Module-Basesystem15-SP1-Pool for x86_64 SMRBS 4.0
+* SLE-Module-Basesystem15-SP1-Updates for x86_64 SMRBS 4.0
+* SLE-Module-Server-Applications15-SP1-Pool for x86_64 SMRBS 4.0
+* SLE-Module-Server-Applications15-SP1-Updates for x86_64 SMRBS 4.0
+* SLE-Product-SUSE-Manager-Retail-Branch-Server-4.0-Updates for x86_64
+. With this activation key created, continue with step 3 of the {susemgrproxy} registration at xref:installation:proxy-registration[].
```

Comment:
I can see this going awry. Can we organise this differently to avoid the "step 3" bit?

Reply:
I can copy paste the rest of proxy registration procedure. Branch Server is essentially Proxy, just with different channels. The configuration is exactly the same.

Reply:
Hrm, I'm not a fan of duplication, either. Can we split the first two steps off for branch server and proxy, then make the remainder for both?

Reply:
Or, list the differences for each in one procedure

Reply:
The difference to proxy configuration is that different channels are used for branch servers.

So I can start the procedure with:

> Branch server is based on {susemgrproxy} and the registration procedure is almost the same.
> Use this procedure to create activation key for the branch server. Then use created activation key in usual {susemgrproxy} registration procedure as described at xref:installation:proxy-registration[].

Or finish the procedure with:

> Use this activation key in {susemgrproxy} registration as instructed at xref:installation:proxy-registration[].